### ISSUE TITLE
setup primp on armv7 / py 3.10 ... any ideas ?

### ISSUE THREAD

**Issue by manatlan** [deedy5/primp#50]
any ideas ?
```
Building wheels for collected packages: aiohttp, MarkupSafe, primp
  Building wheel for aiohttp (pyproject.toml) ... done
  Created wheel for aiohttp: filename=aiohttp-3.11.7-cp310-cp310-linux_armv7l.whl size=1414896 sha256=d9e55595e0841c527752acfc4c3e5eb1b4eebda3a4a272019e0db4e0df230c46
  Stored in directory: /home/pi/.cache/pip/wheels/30/b5/dc/4688f9dcdb7b52f57a5197868e86041f196889ca1a263874ed
  Building wheel for MarkupSafe (pyproject.toml) ... done
  Created wheel for MarkupSafe: filename=MarkupSafe-3.0.2-cp310-cp310-linux_armv7l.whl size=20934 sha256=ef190a5cfdd212dbfde0d3530c0b7bfff479754488800876ee0be1b8397c5b82
  Stored in directory: /home/pi/.cache/pip/wheels/6b/a7/76/330d3979389a2d294838832f620ac3fd32b4ce50d1524c1e29
  Building wheel for primp (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for primp (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [13 lines of output]
      Running `maturin pep517 build-wheel -i /usr/bin/python3 --compatibility off`
      📦 Including license file "/tmp/pip-install-u1fwnlou/primp_2f08d6335e164579b4611174f5e39741/LICENSE"
      🔗 Found pyo3 bindings with abi3 support for Python ≥ 3.8
      🐍 Not using a specific python interpreter
      📡 Using build options features from pyproject.toml
      error: package `rquest v0.27.7` cannot be built because it requires rustc 1.80 or newer, while the currently active rustc version is 1.75.0
      Either upgrade to rustc 1.80 or newer, or use
      cargo update rquest@0.27.7 --precise ver
      where `ver` is the latest version of `rquest` supporting rustc 1.75.0
      💥 maturin failed
        Caused by: Failed to build a native library through cargo
        Caused by: Cargo build finished with "exit status: 101": `env -u CARGO PYO3_ENVIRONMENT_SIGNATURE="cpython-3.10-64bit" PYO3_PYTHON="/usr/bin/python3" PYTHON_SYS_EXECUTABLE="/usr/bin/python3" "cargo" "rustc" "--features" "pyo3/extension-module" "--message-format" "json-render-diagnostics" "--manifest-path" "/tmp/pip-install-u1fwnlou/primp_2f08d6335e164579b4611174f5e39741/Cargo.toml" "--release" "--lib"`
      Error: command ['maturin', 'pep517', 'build-wheel', '-i', '/usr/bin/python3', '--compatibility', 'off'] returned non-zero exit status 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for primp

```

**Comment by deedy5** [comment]
fixed in v0.8.1

**Comment by deedy5** [comment]
@manatlan If you just need to ask a couple questions to AI, give it a try: 
`ddgs chat` 
This is the CLI interface to the duckduckgo_search.
